tests/fixtures/conftest.py

```diff
@@ -23,3 +24,12 @@ def vocabularies(system_process_identity):
         [pathlib.Path(__file__).parent / "data"],
         'vocabularies.yaml',
     )
+
+
+@pytest.fixture(scope="function")
+def admin_role(db):
+    """Create an admin role."""
+    role = Role(name="admin")
```

Comment:
can use this roles? https://github.com/inveniosoftware/invenio-access/blob/87dde7f6b43bda41ec2115ef4a358442d8136703/invenio_access/admin.py#L159

Reply:
:thinking: I don't think so... we just need a pure role in the database named "admin" like the `invenio-cli services setup` gives us:

```python
            CommandStep(
                cmd=['pipenv', 'run', 'invenio', 'roles', 'create', 'admin'],
                env={'PIPENV_VERBOSITY': "-1"},
                message="Creating admin role..."
            )
```